src/client/editor/components/BaseComponent.js

```diff
@@ -21,6 +21,13 @@ export default class BaseComponent {
     this.props[propertyName] = value;
   }
 
+  updateResourceValidation(target, value) {
+    const resourcesValidation = !this._object.resourcesValidation ? {} : this._object.resourcesValidation;
+    resourcesValidation[target] = value;
+    //this.updateProperty("_resourcesValidation", resourcesValidation);
+    this._object.resourcesValidation = resourcesValidation;
+  }
```

Comment:
I don't think `BaseComponent` should have a method called `updateResourceValidation`. Most components will not have resources associated with them. It also does not make sense to store `resourcesValidation` on `_object` -- it is associated with the component, specifically the component's property, not with the `_object`.